bpf/Makefile

```diff
@@ -67,9 +67,11 @@ LB_OPTIONS = \
 
 # These options are intended to max out the BPF program complexity. it is load
 # tested as well.
-MAX_LB_OPTIONS = -DENABLE_IPV4 -DENABLE_IPV6 -DENCAP_IFINDEX=1 \
-	-DENABLE_HOST_SERVICES_TCP -DENABLE_HOST_SERVICES_UDP \
-	-DENABLE_IPSEC
+MAX_BASE_OPTIONS = -DSKIP_DEBUG -DENABLE_IPV4 -DENABLE_IPV6 \
```

Comment:
nit: it's not worth redoing the series for this, but if you redo it for other reasons, please consider using one option per line which would  make subsequent diffs clearer.

Reply:
I'm not sure this is worth it. On the one hand, I agree that it would help diffing when there are a lot of changes as done here. On the other hand, (1) future extensions of the list of options are very likely to just add to the end, making any diffing easy regardless of the format, and (2) we could easily end up with a long page of options to scroll down.